### ISSUE TITLE
Loading issue

### ISSUE THREAD

**Issue by AeSix** [OopsieWoopsie/Spicord#8]
So, trying to get BanAnnouncer working... 
It gave me this error:
```
02:38:54 [INFO] [Spicord] Loaded 1 bots, 0 disabled.
02:38:54 [INFO] [Spicord] [DEBUG] [Loader] Loaded library 'JDA-3.8.3_460-withDependencies.jar'.
02:38:54 [INFO] [Spicord] Registered addon 'Server Information' (spicord::info) by OopsieWoopsie
02:38:54 [INFO] [Spicord] Registered addon 'Plugin List' (spicord::plugins) by OopsieWoopsie
02:38:54 [INFO] [Spicord] Registered addon 'Player List' (spicord::players) by OopsieWoopsie
02:38:54 [INFO] [Spicord] [DEBUG] Disabling JDA's messages...
02:38:54 [INFO] [Spicord] [DEBUG] Successfully disabled JDA's messages.
02:38:54 [INFO] [Spicord] Starting the bots...
02:38:54 [INFO] [Spicord] Starting bot 'default'.
02:38:54 [SEVERE] Exception in thread "Thread-38"
02:38:54 [SEVERE] java.lang.LinkageError: loader constraint violation: when resolving method "org.slf4j.impl.StaticMDCBinder.getMDCA()Lorg/slf4j/spi/MDCAdapter;" the class loader (instance of net/md_5/bungee/api/plugin/PluginClassloader) of the current class, org/slf4j/MDC, and the class loader (instance of net/md_5/bungee/api/plugin/PluginClassloader) for the method's defining class, org/slf4j/impl/StaticMDCBinder, have different Class objects for the type org/slf4j/spi/MDCAdapter used in the signature
02:38:54 [SEVERE]     at org.slf4j.MDC.bwCompatibleGetMDCAdapterFromBinder(MDC.java:99)
02:38:54 [SEVERE]     at org.slf4j.MDC.<clinit>(MDC.java:108)
02:38:54 [SEVERE]     at net.dv8tion.jda.core.entities.impl.JDAImpl.login(JDAImpl.java:207)
02:38:54 [SEVERE]     at net.dv8tion.jda.core.JDABuilder.build(JDABuilder.java:913)
02:38:54 [SEVERE]     at eu.mcdb.spicord.bot.DiscordBot.startBot(DiscordBot.java:105)
02:38:54 [SEVERE]     at eu.mcdb.spicord.bot.DiscordBotLoader$1.run(DiscordBotLoader.java:40)
02:38:54 [SEVERE]     at java.lang.Thread.run(Thread.java:748)
```
Versions:
  BanAnnouncer: 1.0.2 F1  (From Github)
  Spicord: 1.0.4-SNAPSHOT (from SpigotMC)
  AdvancedBans 2.1.5 (from SpigotMC)
BungeeCord 1.14 (build 1414)

note: I have no 1.14 back-end servers, am not using Waterfall/Travertine, etc, just md_5's BungeeCord.  I have no other errors in the logs.

Edit, adding info:
$ java -version
java version "1.8.0_201"
Java(TM) SE Runtime Environment (build 1.8.0_201-b09)
Java HotSpot(TM) 64-Bit Server VM (build 25.201-b09, mixed mode)

**Comment by AeSix** [comment]
I am running Ubuntu 18.04, mostly updated. 
I'm not using any plugin for discord at all on Bungee. 

It would be lovely to have this working, unfortunately, that's the only thing I've seen in the log when starting up Bungee, so I'm afraid I can't be of a whole lot of use to you in that.

**Comment by OopsieWoopsie** [comment]
I will stop working on other things and try to fix this asap, I have enough information for fix it.
Thanks for reporting this issue. ^^

Have a good day (night?) !

**Comment by OopsieWoopsie** [comment]
It should be fixed now, please try it and tell me if it works.

https://github.com/OopsieWoopsie/Spicord/releases/tag/1.1.0-SNAPSHOT

**Comment by AeSix** [comment]
I will first thing in the morning. Thank you! 
(I'm aboit to passout, otherwise would do it now)

**Comment by AeSix** [comment]
Seems to be working good! 
I was able to get AdvancedBan 2.1.6 to build, and so updated that.  Using 1.1.0 of Spicord and the same 1.0.2 BanAnnouncer.  Updated Bungee to build #1416 as well. 

Was able to get the bot loaded, replying to commands, and relaying punishments! 

Since this issue seems to be resolved, I'll go ahead and close the ticket. 

Thank you very much for the quickness! (even a week late is quicker than some)
